### DISCUSSION TITLE
unity 2022.3.6f1

### DISCUSSION THREAD

**Discussion by hyde4280** [esperecyan/VRMConverterForVRChat#63] [Ideas]
unity 2022.3.6f1対応何かしないとダメでしょうか？　少なくとも私は使えないみたいです。

**Comment by hyde4280** [comment]
VCCで 2019→2022 migrate したからのようで、新規プロジェクトだったらwarningは出ますが動作はするようでした。

**Reply by esperecyan** [reply]
ご報告有り難うございます。issueを作成しました
https://github.com/esperecyan/VRMConverterForVRChat/issues/64
